Makefile

```diff
@@ -137,6 +137,17 @@ test.neon::
 		$(REGRESSION_TEST_MODE)
 	@ echo "herd7 AArch64 NEON instructions tests: OK"
 
+test:: test.sve
+test.sve::
+	@ echo
+	$(HERD_REGRESSION_TEST) \
+		-herd-path $(HERD) \
+		-libdir-path ./herd/libdir \
+		-litmus-dir ./herd/tests/instructions/AArch64.sve \
+		-variant neon \
```

Comment:
Should we introduce a synonym for -variant neon, something like "vector". I find it counter intuitive that we need to enable -variant neon for sve.

Reply:
Don't we need two different variants as neon and sve are different?

Reply:
We could have two different variants `sve` and `neon` but the underlying functionality that is enabled is the same. So either we introduce `sve` and we make it a synonym to `neon` or we introduce a common one `vector`. The only solution I think would be confusing would be to ask someone to enable `-variant neon` to run an SVE test.

Reply:
But in any case, the way Vladimir has implemented this change, and I think that's the right approach, -variant neon still works.

Reply:
But what about generators? Is it acceptable to  generate sve  code with `-variant neon`?

Reply:
ah sorry I thought we were referring to `herd7`. For the generator I am not sure, what do use `-variant neon` for? Or why would we need to introduce -variant sve? If a user specifies an SVE or a Neon relaxation, shouldn't diyone7 just produce the test?

Reply:
Well I'd expect `-variant neon` to accept a strict subset of the annotations accepted by `-variant sve`. More precisely, with head version we have:
```
$ diyone7 -arch AArch64 -variant neon -show annotations
 X XL XA XAL I L Q A PaI PaN Pa Ne4i Ne3i Ne2i Ne4 Ne3 Ne2 Ne1 NePaN NePa NeP NeL NeQ
```
While with this PR, we have:
```
$ /opt/WIP/bin/diyone7 -arch AArch64 -variant neon -show annotations
 X XL XA XAL I L Q A PaI PaN Pa Sv4i Sv3i Sv2i Sv1 SvV Ne4i Ne3i Ne2i Ne4 Ne3 Ne2 Ne1 NePaN NePa NeP NeL NeQ
```
Obviously some annotations have been introduced: `Sv4i Sv3i Sv2i Sv1 SvV`. It would perhaps be more natural if they were available only with a specific `-variant sve` option. Notice that I have no idea whether `Sv...` annotations lead to code that is neon compatible or not. I am just considering the names themselves.

Reply:
Ok, I see diyone7 will use `-variant sve|neon` to decide which annotations and relaxations it will list.

Then, it sounds like the cleanest option would be to have two different variants `sve` and `neon`. For herd7, they would be synonyms, but for diyone7, they would enable different annotations. Also, we drop `vector`. `vector` makes sense for herd7 but it doesn't make sense for `diyone7` and it's not worth having different variants for the two. Do you agree?

Reply:
Not only which annotation it would list, but more significantly, which annotation it would accept.

Then, `vector` could exist for all tools as a synonymous of "the most advanced vector extension"... ?

BTW, can someone try running sve tests on actual hardware?  The appropriate gcc compile option should be something like `-march=armv8.2-a+sve`. It may be useful to define a litmus configuration file similar to `armv8.1.cfg`.